subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossBuildPerformanceTest.groovy

```diff
@@ -30,13 +30,15 @@ import org.gradle.performance.results.GradleProfilerReporter
 import org.gradle.performance.results.WritableResultsStore
 import org.gradle.test.fixtures.file.CleanupTestDirectory
 import org.gradle.test.fixtures.file.TestNameTestDirectoryProvider
+import org.gradle.util.UsesNativeServices
 import org.junit.Rule
 import spock.lang.Specification
 
 import static org.gradle.performance.results.ResultsStoreHelper.createResultsStoreWhenDatabaseAvailable
 
 @CompileStatic
 @CleanupTestDirectory
+@UsesNativeServices // The TAPI invoker uses native services
```

Comment:
Why does it do that? :o

Reply:
For console output or for obtaining the PID or the current process? All the clients use native platform I think.

Reply:
👍